enterprise/views.py

```diff
@@ -74,10 +76,11 @@ def verify_edx_resources():
         get_enterprise_customer_for_request,
         get_real_social_auth_object,
         quarantine_session,
-        lift_quarantine
+        lift_quarantine,
+        get_partial_pipeline,
     )
     if any(method is None for method in required_methods):
-        raise NotConnectedToEdX(_('Methods in the Open edX platform necessary for this view are not available.'))
+        raise NotConnectedToOpenEdX(_('Methods in the Open edX platform necessary for this view are not available.'))
```

Comment:
also worth updating the message here to be more specific about the missing dependencies.